clang/test/CIR/CallConvLowering/AArch64/aarch64-cc-structs.c

```diff
@@ -73,3 +73,28 @@ GT_128 ret_gt_128() {
   GT_128 x;
   return x;
 }
+
+// CHECK: cir.func {{.*@pass_lt_64}}(%arg0: !u64
```

Comment:
Can you please add tests for the LLVM output as well? It's important to have that test because it's either equivalent to OG or should have a comment on what needs to be done later. Ultimately matching OG is the ABI goal.

Reply:
added!